src/schema/v2/me/myCollectionInfo.ts

```diff
@@ -33,6 +37,30 @@ export const myCollectionInfoFields = {
     type: new GraphQLNonNull(GraphQLInt),
     resolve: ({ artists_count }) => artists_count,
   },
+  collectedArtistsConnection: {
+    description: "A connection of artists in the users' collection",
+    type: artistConnection.connectionType,
+    args: pageable({
+      page: { type: GraphQLInt },
+      size: { type: GraphQLInt },
+    }),
+    resolve: async (_root, args, context) => {
+      const { collectionArtistsLoader } = context
+
```

Comment:
```suggestion
    resolve: async (_root, args, { collectionArtistsLoader }) => {
```

Reply:
when I do it this way the typing for `fields` in line 66 breaks and I couldn't fix it 🤷

Reply:
Ah, because authenticated loaders can be undefined. Let's leave it like this then :)